packages/next/src/server/config.ts

```diff
@@ -471,6 +471,13 @@ function assignDefaults(
     silent
   )
 
+  warnOptionHasBeenDeprecated(
```

Comment:
should this be `warnOptionHasBeenMovedOutOfExperimental`?

Reply:
I made a new one cause `warnOptionHasBeenMovedOutOfExperimental` will say the flag is moved to xxx, mostly for the flags that are still remained. Here we delete the flag, similar others like `swcMinify`. So we add a warning for users that they can delete the unexisted flag